compose/backend.yml

```diff
@@ -165,23 +162,25 @@ services:
       - ccd-docker-ccd-shared-database-data:/var/lib/postgresql/data
 
   shared-db:
-    image: hmcts.azurecr.io/hmcts/sidam/sidam-idm-pg:1.0.0
+    image: "hmctsprivate.azurecr.io/idam/shared-db:latest"
     ports:
       - 5051:5432
+    volumes:
+      - ccd-docker-idam-shared-database-data:/var/lib/postgresql/data  
```

Comment:
What's the rationale of this change? please avoid changes not related to the ticket scope.
We already have a branch where we introduced a volume to sidam db. That branch is currently under testing.

Reply:
@mario-paniccia  just thought of improving this, sorry didn't see other PR, will revert